components/cop_codegen/src/rpn_function.rs

```diff
@@ -255,6 +258,27 @@ fn parse_arg_type(arg: &FnArg) -> Result<TypePath> {
     Ok(eval_type.clone())
 }
 
+fn check_attr(meta: &[Ident]) -> Result<()> {
```

Comment:
Check the number of parameters of the function is at least the attribute count instead.